### ISSUE TITLE
Point material configuration

### ISSUE THREAD

**Issue by jconnolly-bond** [nytimes/three-loader-3dtiles#35]
Hi there, 
First of all thank you for the great lib.

I was wondering if it would be possible to expose the material used by points or allow for its properties to be updated?

The main use case I have in mind is being able to change the opacity of all the points being rendered dynamically.

Thanks very much.

**Comment by Avnerus** [comment]
Hi! 
Thanks for the feedback.
This sounds like a worthy feature, but we haven't had the need for it so far. Would you be interested in implementing it via a pull request?
The implementation should be quite simple since we already have a `material` option, it's just used only for meshes.
I think all that is needed is to pass `options` to `createPointNodes` and use `option.material` if it exists, instead of our stock one in this [line](https://github.com/nytimes/three-loader-3dtiles/blob/8bb89faca93672584449a2efa9d193207e0823c6/src/index.ts#L558). Would be great if you could give it a go!

/Avner

**Comment by jconnolly-bond** [comment]
Sure I can create a PR, not sure when but I can do.

Looking at the code, am I right in thinking that setting the material within `createPointNodes` would only affect newly loaded tiles? It looks like it may have to be implemented with a method like `runtime.setPointCloudOpacity(opacity)` in order to support dynamically adjusting it for all visible points. Any thoughts?

Thanks very much.

**Comment by Avnerus** [comment]
Hi, I think it should still allow you to update the opacity in runtime if you use your own uniforms and shader. So usage would be something like:
```
  const myMaterial = new ShaderMaterial({
    uniforms: myUniforms
    vertexShader: myVS
    fragmentShader: myFS
    transparent: true,
  });

/* Inside the loader
  const tileContent = new Points(geometry, options.material);
*/

const result = await Loader3DTiles.load(
  {
    url:
     ...
    options: {
       material: myMaterial
    }
  }


renderLoop() {
    myMaterial.uniforms.alpha.value = myOpacity;
}
```
That should dynamically change the opacity of all points since they all use the same shader. 
If you need a separate shader/opacity per tile it's a bit more complex, but still possible using the `shaderCallback` option.

Hope this helps!

**Comment by Avnerus** [comment]
This is now released in v1.1.5.
Thank you for the PR!
